### ISSUE TITLE
GGMAX 25 MAR Version - Request - Let users place objects outside of editable area

### ISSUE THREAD

**Issue by cybernescence** [TheGameCreators/GameGuruRepo#2653]
Sometimes game vistas require that assets can be placed where players cannot reach, such as buildings etc on horizon to evoke the game style required. The terrain is great/amazing, but I think being able to see other user assets beyond this editable area would also benefit game dev.

(Specifically in my case I was also developing a flying prototype that let the player go beyond the editable area and use the brilliant 'endless' terrain system views - and also to land on user placed assets out in the 'wild blue' - can't do this now with latest restrictions).

I do understand why it has been put in place, perhaps can be a toggle off setting somewhere?

Cheers.

**Comment by Mt-Chiliad-UFO** [comment]
Yes, makes sense. This sounds like a good way any designer could set a "preview" for the next level; put some assets / scenery near level complete trigger (though outside the current level) that suggest what the next level might entail. Is it a valley full of ruffians, or a stout mountain topped off by concertina fences?!

**Comment by Necrym59** [comment]
Defeats the purpose of having an Editable area, just make yours a bit smaller then populate the edge area behind a win zone most    level designers do this otherwise there is no end to how far you want editible area to be, it cant be infinite.

**Comment by cybernescence** [comment]
That limits the sweeping vistas and objects in the air at distance etc. Total shame to waste all that work on the terrain views by limiting what else can be seen on it, around it or above it.

**Comment by Necrym59** [comment]
Well yess and no this isnt a flight simulator, what your after is decorative distance related. When they get around to streaming textures for objects like thay have done for trees and terrain, anythings possible then and ill think some of what you ask wiould then be viable.
